features/device_key/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+target_include_directories(mbed-os
+	PUBLIC
+		source
+)
+
+target_sources(mbed-os
+	PRIVATE
+		source/DeviceKey.cpp
```

Comment:
just wondering about the formating, current cmakes have it as

```
target_sources(project PRIVATE
    files.c
    file2.cpp
    ....
)
```

Reply:
I was watching a tutorial video from a CMake expert and the thinking behind this indentation is to separate `PRIVATE` and `PUBLIC` files for readability the same way as it is commonly indented in C++ classes.

```
target_sources(project
    PRIVATE
        file1.cpp
        file2.cpp
    PUBLIC
        file3.cpp
)
```

I did not think we have already established a style to follow.

Reply:
Makes sense. We unify , probably this could be a way then.